lib/index.js

```diff
@@ -51,7 +51,18 @@ Comscore.prototype.loaded = function() {
  * @param {Object} page
  */
 
-Comscore.prototype.page = function() {
+Comscore.prototype.page = function(page) {
   var comScoreOptions = { c1: this.options.c1, c2: this.options.c2 };
+  var beaconParamMap = this.options.beaconParamMap;
+  var properties = page.properties();
+
+  Object.keys(this.options.beaconParamMap).forEach(function(property) {
```

Comment:
Small nitpick for later, but a bit odd that we define this on 56 and use the full name here.

Reply:
Indeed, I'll fix this.

Btw I only created a variable because of `this` nonsense.